sirepo/package_data/static/js/raydata.js

```diff
@@ -741,6 +746,16 @@ SIREPO.app.directive('scanSelector', function() {
                 $scope.selectAllColumns = ! $scope.selectAllColumns;
             };
 
+            $scope.unselectAllScans = () => {
+                Object.keys(appState.models.selectedScans.uids).forEach((u) => {
+                    delete appState.models.selectedScans.uids[u];
+                });
+                for (let i = 0; i < $scope.scans.length; i++) {
```

Comment:
This is up for debate but I prefer functional style. So, that would be

`$scope.scans.map(s => $scope.toggleScanSelection(s, false))`